src/main/java/org/symphonyoss/symphony/messageml/elements/Element.java

```diff
@@ -638,8 +647,15 @@ void assertNoText() throws InvalidInputException {
    * Check that the element's children are limited to phrasing content.
    */
   void assertPhrasingContent() throws InvalidInputException {
```

Comment:
We could simply consider that `<pre>` is a phrasing content

Reply:
But this would impact all other elements: Bold, Emoji, Header, Italic and Preformatted. Does it sound good to you?

Reply:
Let's perhaps check with FE team if `<b><pre>Hello</pre></b>`, `<i><pre>Hello</pre></i>` or `<pre><pre>Hello</pre></pre>` is valid. For me it should